quickstart/utils/ingestion/ingestion.go

```diff
@@ -0,0 +1,43 @@
+// Package ingestion - in charge of ingesting the given data - based on the configuration file.
+package ingestion
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-kusto-go/kusto/data/errors"
+	"github.com/Azure/azure-kusto-go/kusto/ingest"
+	"github.com/Azure/azure-kusto-go/quickstart/utils"
+	"time"
+)
+
+// WaitForIngestionToComplete Halts the program for WaitForIngestSeconds, allowing the queued ingestion process to complete
+func WaitForIngestionToComplete(waitForIngestSeconds int) {
+	fmt.Printf("Sleeping %d seconds for queued ingestion to complete. Note: This may take longer depending on"+
+		" the file size and ingestion batching policy.\n", waitForIngestSeconds)
+
+	for i := 0; i <= waitForIngestSeconds; i++ {
+		fmt.Print("\r")
+
+		fmt.Print(i)
+		for j := 0; j <= i; j++ {
+			fmt.Print(".")
+		}
+		time.Sleep(time.Second)
+	}
+}
+
+// CreateIngestionOptions creates ingestion options: ingestion mapping and anything else necessary
+func CreateIngestionOptions(mappingName string, dataFormat ingest.DataFormat) []ingest.FileOption {
```

Comment:
This is only used once and doesn't really save any lines, do you really need it?

Reply:
Not necessarily, was mostly done to maintain a similar structure to other SDKs, and to allow reusability.  should i remove it then?

Reply:
I think we can remove it for now